### DISCUSSION TITLE
Option to use current timestamp

### DISCUSSION THREAD

**Discussion by vacadam** [megahertz/electron-log#446] [Q&A]
Is there a predefined option to use current timestamp?

I tried to do custom format:

```
log.transports.file.format = (data) => {

  const timestamp = Date.now();


  return `<log4j:event logger="Application.XY" timestamp="${timestamp}" level="${data.level}" thread="1">
<log4j:message>${data.message}</log4j:message>
</log4j:event>`;
};
```

This indeed logs current timestemp, but it creates two issues:

- ${data.message} outputs [object Object] instead of the actual log message/text
- formatting of the output is broken and it creates empty space after each character

```
< l o g 4 j : e v e n t   l o g g e r = " A p p l i c a t i o n "   t i m e s t a m p = " 1 7 3 3 9 9 1 1 7 3 2 5 1 "   l e v e l = " i n f o "   t h r e a d = " 1 " > 
 < l o g 4 j : m e s s a g e > [ o b j e c t   O b j e c t ] < / l o g 4 j : m e s s a g e > 
 < / l o g 4 j : e v e n t >
```

**Comment by megahertz** [comment]
Check the actual data types https://github.com/megahertz/electron-log/blob/843de37f0451e69b3b99de5296d81b18ed693fd3/src/index.d.ts#L9

First of all, run console.log(data) for debugging. The result should be an array of strings.
